linux_os/guide/system/software/disk_partitioning/partition_for_tmp/rule.yml

```diff
@@ -12,7 +12,7 @@ rationale: |-
     Placing <tt>/tmp</tt> in its own partition enables the setting of more
     restrictive mount options, which can help protect programs which use it.
 
-severity: low
+severity: medium
```

Comment:
RHEL7 has this as a low.

Reply:
We can always make the severity adjustments on the profiles, that is definitely a less disruptive change.

Reply:
Done